src/Sarif/Schemata/sarif-schema.json

```diff
@@ -221,8 +237,13 @@
         },
 
         "binary": {
-          "description": "MIME Base64-encoded content from a binary artifact, or from a text artifact in its original encoding.",
-          "type": "string"
+            "description": "MIME Base64-encoded content from a binary artifact, or from a text artifact in its original encoding.",
+            "type": "string"
+        },
+
+        "rendered": {
+            "description": "TBD",
```

Comment:
>TBD [](start = 28, length = 3)

An alternate rendered representation of the artifact (e.g., a decompiled representation of a binary region).